.github/CODEOWNERS

```diff
@@ -1479,6 +1479,27 @@ x-pack/plugins/cloud_integrations/cloud_full_story/server/config.ts @elastic/kib
 #CC# /x-pack/plugins/translations/ @elastic/kibana-localization @elastic/kibana-core
 
 # Kibana Platform Security
+# security
+/x-pack/test/functional/es_archives/rule_exceptions @elastic/kibana-security # Assigned per https://github.com/elastic/kibana/pull/199797#discussion_r1838166357
+/x-pack/test_serverless/functional/test_suites/observability/role_management @elastic/kibana-security
+/x-pack/test/functional/config_security_basic.ts @elastic/kibana-security
+/x-pack/test/functional/page_objects/user_profile_page.ts @elastic/kibana-security
+/x-pack/test/functional/page_objects/space_selector_page.ts @elastic/kibana-security
+/x-pack/test/functional/page_objects/security_page.ts @elastic/kibana-security
+/x-pack/test/functional/page_objects/role_mappings_page.ts @elastic/kibana-security
+/x-pack/test/functional/page_objects/copy_saved_objects_to_space_page.ts @elastic/kibana-security # Assigned per https://github.com/elastic/kibana/pull/39002
+/x-pack/test/functional/page_objects/api_keys_page.ts @elastic/kibana-security
+/x-pack/test/functional/page_objects/account_settings_page.ts @elastic/kibana-security
+/x-pack/test/functional/apps/user_profiles @elastic/kibana-security
+/x-pack/test/common/services/spaces.ts @elastic/kibana-security
+/x-pack/test/api_integration/config_security_*.ts @elastic/kibana-security
+/x-pack/test/functional/apps/api_keys @elastic/kibana-security
+/x-pack/test/ftr_apis @elastic/kibana-security
+/test/server_integration/services/supertest.js @elastic/kibana-security # Assigned per https://github.com/elastic/kibana/pull/53810
+/test/server_integration/http/ssl @elastic/kibana-security # Assigned per https://github.com/elastic/kibana/pull/53810
+/test/server_integration/config.base.js @elastic/kibana-security # Assigned per https://github.com/elastic/kibana/pull/53810
+/test/server_integration/__fixtures__ @elastic/kibana-security # Assigned per https://github.com/elastic/kibana/pull/53810
```

Comment:
Looks like most of these were introduced by @elastic/kibana-core in #19261. That said, there is certainly some overlap with our team. I think we'd be ok will owing or having joint ownership of some of these (ssl folders & fixtures folder to be specific), but we should check with core first.

Reply:
@elastic/kibana-core can someone weigh in here pretty please?

Reply:
Asked in slack

Reply:
yeah I think on the whole security would be a better reviewer if we do ever change anything SSL related so it'd make sense to me for them to own it.

Core could be a co-owner of `/test/server_integration/config.base.js` and I think it makes the most sense for us to own `test/server_integration/http/ssl_redirect/index.ts`

I think security should also own the other ssl* folders like `ssl_with_p12` and `ssl_with_p12_intermediate`

Reply:
> I think security should also own the other ssl* folders like ssl_with_p12 and ssl_with_p12_intermediate

++

Reply:
https://github.com/elastic/kibana/pull/199795/commits/5f41777673f17cef81611c40d832fc42ba65a2de && https://github.com/elastic/kibana/pull/199795/commits/894a8ede3f9d0398c5af56bf5a82654a9bc0610b